.github/workflows/library_interop_tests.yml

```
@@ -0,0 +1,183 @@
+# This workflow performs interoperability tests across the supported runtimes of the MPL.
+name: Library Interoperability Tests
+
+on:
+  workflow_call:
+    inputs:
+      dafny:
+        description: "The Dafny version to run"
+        required: true
+        type: string
+
+jobs:
+  generateEncryptVectors:
+    strategy:
+      matrix:
+        library: [TestVectorsAwsCryptographicMaterialProviders]
+        os: [
+            # TODO just test on mac and ubuntu for now
+            # windows-latest,
+            ubuntu-latest,
+            macos-latest,
+          ]
+        languages: [java, net]
```

```suggestion
        language: [java, net]
```
nit; when I wrote the `if` conditional below, I realized it makes more sense if matrix variables are singular